gwdetchar/tests/test_condor.py

```diff
@@ -46,7 +45,7 @@ def test_accounting_epoch(gps, epoch):
 def tagfile():
     name = tempfile.mktemp()
     with open(name, 'w') as tmp:
-        json.dump({'groups': ['tag1', 'tag2', 'tag3']}, tmp)
+        tmp.write("* tag1 found\n* tag2 found\n* tag3 found\n")
```

Comment:
This change in formatting (not the change in how it's saved) versus the old formatting is intentional, right?

Reply:
Yes, the file format is completely different now. The list of valid tags is not a json file but rather an ASCII file with entries like: `* tag1 found`

Reply:
Never mind - I just noticed that it's in the PR's actual description.